### PR TITLE
chore: add code example with react

### PR DESCRIPTION
**Pull request by awx-josh-fu**
## Description

Add code example of how to user login widget with react

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Add code example

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: examples/react/pnpm-lock.yaml [hunk 1]
```diff
@@ -0,0 +1,1919 @@
+lockfileVersion: 5.3
```
Comment (CharlieLangCL) [inline examples/react/pnpm-lock.yaml:1]: maybe no need to lock versions for example ？

File: examples/react/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Airwallex Login Widget React Example
+
+This is an example of how to use the Airwallex Login Widget with React.
+
+## How to start
+
+`pnpm install && pnpm dev`
```
Comment (CharlieLangCL) [inline examples/react/README.md:7]: could we put a screen shoot in the readme so user can compare the local running example with it ?

File: examples/react/pnpm-lock.yaml (hunk 1 above)
Comment (awx-josh-fu) [reply examples/react/pnpm-lock.yaml:1]: Should I use `npm` instead?

File: examples/react/README.md (hunk 2 above)
Comment (awx-josh-fu) [reply examples/react/README.md:7]: Of course

File: examples/react/pnpm-lock.yaml (hunk 1 above)
Comment (CharlieLangCL) [reply examples/react/pnpm-lock.yaml:1]: maybe 🤔 a good choice to keep the example simple a bit

**Review by CharlieLangCL — APPROVED** [review]

**Review by airwallex-guitingting — APPROVED** [review]

**Pull request merged**